core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -28,7 +30,6 @@
  * @property string         $field
  * @property string         $inputName
  * @property string         $palette
- * @property object|null    $activeRecord
```

Comment:
As long as the property is still there, we should keep the annotation.

Reply:
Then you don't see that it's deprecated. There's no way to deprecate an `@property` annotation or at least I couldn't find it. Neither could @ausi.

Reply:
I see.